### PR TITLE
motif finding should avoid name conflict when matching with anonymous edge

### LINKED ISSUE #65: motif finding should avoid name conflict when matching with anonymous edge
Currently it is using `__tmp` as column name which could theoretically conflict with an actual column with that name.
See #50

### PR DESCRIPTION
**Pull request by felixcheung**
Fix #65 with randomly generated name
@jkbradley

### CODE REVIEW

File: src/main/scala/org/graphframes/GraphFrame.scala [hunk 1]
```diff
@@ -32,6 +32,8 @@ import org.apache.spark.storage.StorageLevel
 import org.graphframes.lib._
 import org.graphframes.pattern._
 
+import scala.reflect.runtime.universe.TypeTag
```
Comment (jkbradley) [inline src/main/scala/org/graphframes/GraphFrame.scala]: Let's follow Spark style: This should go between the Java & the org.* imports.

**Comment by jkbradley** [comment]
This looks good, but could you please add a little unit test which fails before your fix?  Thanks!

**Comment by felixcheung** [comment]
Done. Thanks for the feedback!

**Comment by jkbradley** [comment]
LGTM, thanks again!

**Pull request merged**